browser/resources/settings/brave_appearance_page/content.ts

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import '../settings_shared.css.js'
+import '../settings_vars.css.js'
+
+import {PrefsMixin, PrefsMixinInterface} from 'chrome://resources/cr_components/settings_prefs/prefs_mixin.js';
+import {I18nMixin, I18nMixinInterface} from 'chrome://resources/cr_elements/i18n_mixin.js'
+import {PolymerElement} from 'chrome://resources/polymer/v3_0/polymer/polymer_bundled.min.js'
+import {DropdownMenuOptionList, SettingsDropdownMenuElement} from '/shared/settings/controls/settings_dropdown_menu.js';
+
+import {BaseMixin} from '../base_mixin.js';
+import {loadTimeData} from '../i18n_setup.js'
+import {routes} from '../route.js';
+import {Router} from '../router.js';
+
+import {AppearanceBrowserProxy, AppearanceBrowserProxyImpl} from '../appearance_page/appearance_browser_proxy.js';
+import {getTemplate} from './content.html.js'
+
+/**
+ * This is the absolute difference maintained between standard and
+ * fixed-width font sizes. http://crbug.com/91922.
+ */
+const SIZE_DIFFERENCE_FIXED_STANDARD: number = 3;
+
+export interface SettingsBraveAppearanceContentElement {
+  $: {
+    defaultFontSize: SettingsDropdownMenuElement,
+    zoomLevel: HTMLSelectElement,
+  };
+}
+
+const SettingsBraveAppearanceContentElementBase =
+    I18nMixin(PrefsMixin(BaseMixin(PolymerElement)));
+
+export class SettingsBraveAppearanceContentElement extends SettingsBraveAppearanceContentElementBase {
```

Comment:
I think we don't miss newly added options as we only copied specific options.
and If copied options code is updated from upstream, maybe we could catch it because we still reuse upstream's prefs, load time data and proxy.